tests/nfa/builder.cc

```diff
@@ -2,6 +2,7 @@
 
 #include <unordered_set>
 #include <fstream>
+#include <cmath>
```

Comment:
Not just <math> or whatever is the C++ equivalent (if there even is one)?

Reply:
It seems that `cmath` is the only library that provides rounding.